packages/components/header/_header-white.scss

```diff
@@ -67,7 +67,7 @@
     }
   }
 
-  .nhsuk-header__navigation-list {
+  .nhsuk-navigation {
```

Comment:
Can you remember why this class name changed? I’d have expected all the css to be scoped within `.nhsuk-header*` but I might have forgotten something?

Reply:
I haven’t changed any class names, just changed which element these styles are applied to.

The classes used in this component are a bit all over the place (as elsewhere). For example, the header contains:

- `nhsuk-navigation-container`
- `nhsuk-navigation`
- `nhsuk-mobile-menu-container`
- `nhsuk-organisation-*`
- `nhsuk-logo`
- `nhsuk-search__*`

If we want to fix this, perhaps something for a different PR?